src/doc.c

```diff
@@ -2325,6 +2327,41 @@ void highlightText(Scope *sc, Dsymbol *s, OutBuffer *buf, size_t offset)
                 }
                 break;
 
+            case '`':
+                if (inBacktick)
+                {
+                    inBacktick = 0;
+                    inCode = 0;
+
+                    OutBuffer codebuf;
+
+                    codebuf.write(buf->data + iCodeStart + 1, i - (iCodeStart + 1));
+                    codebuf.writeByte(0);
+
+                    highlightCode2(sc, s, &codebuf, 0);
+
+                    buf->remove(iCodeStart, i - iCodeStart + 1); // also trimming off the current `
+                    i = buf->insert(iCodeStart, codebuf.data, codebuf.offset);
+                    i = buf->insert(i, ")", 1);
+
+                    break;
+                }
+
+                if (inCode)
+                    break;
+
+                inCode = 1;
+                inBacktick = 1;
+
+                static const char pre[] = "$(D_INLINECODE ";
```

Comment:
I don't agree since it is being highlighted as D code. This is $(EMPHASIS not) just syntax sugar being lowered to another macro $(PARENTHETICAL_ASIDE that would be useless to me btw) - it is another entry point specifically into the D syntax highlighting and code escaping feature.